include/sound/sof.h

```diff
@@ -92,16 +84,13 @@ struct sof_dev_desc {
 	const char *default_fw_path;
 	const char *default_tplg_path;
 
+	/* default firmware name */
+	const char *default_fw_filename;
+
```

Comment:
I think this field should be introduced in a separate commit as it is not part of previous code.

Reply:
done